fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/RuleSet.java

```diff
@@ -265,8 +264,7 @@ public class RuleSet {
                 new ExistentialApply2JoinRule(),
                 new ScalarApply2JoinRule(),
                 new ExistentialApply2OuterJoinRule(),
-                new QuantifiedApply2OuterJoinRule(),
-                new ApplyExceptionRule()
```

Comment:
Apply operator may be processed later？

Reply:
ApplyExceptionRule just for throw exception when meet can't handle subquery, will execute it after all subquery rule